src/components/form/SelectCondition/index.tsx

```diff
@@ -53,35 +43,29 @@ export const SelectCondition = () => {
 
   return (
     <>
-      <Row cols={'1fr'} marginBottomXL>
-        <TitleValue
-          title="Condition Id"
-          titleControl={<TitleControl onClick={openModal}>Select Condition</TitleControl>}
-          value={
-            <>
-              <Textfield
-                error={!!errors.length}
-                onChange={imputHandler}
-                placeholder="Please select a condition..."
-                type="text"
-                value={conditionId}
-              />
-              {loading && (
-                <ErrorContainer>
-                  <InfoMessage>Loading...</InfoMessage>
-                </ErrorContainer>
-              )}
-              {!!errors.length && (
-                <ErrorContainer>
-                  {errors.map((error, i) => (
-                    <Error key={`error${i}`}>{error}</Error>
-                  ))}
-                </ErrorContainer>
-              )}
-            </>
-          }
-        />
-      </Row>
+      <TitleValue
+        title={title}
+        titleControl={<TitleControl onClick={openModal}>Select Condition</TitleControl>}
+        value={
+          <>
+            <Textfield
+              error={!!errors.length}
+              onChange={imputHandler}
```

Comment:
Don't worry about this wording, is already fixed in another PR

Reply:
I don't understand. What should (or shouldn't) I do about this?